.github/workflows/apply_cirun.yml

```diff
@@ -16,7 +16,9 @@ jobs:
     runs-on: ubuntu-latest
     if: contains(github.event.pull_request.labels.*.name, 'ci:ready_to_merge')
     steps:
-      - uses: actions/github-script@v5
+      - name: add ci run
```

Comment:
Please update the file comments to reflect the latest behavior. Maybe something like- "1. ci:ready_to_merge is added to a PR, by the mergify[bot]", would do?

Reply:
The comments as they exist are accurate. ci:ready_to_merge is added to a pr manually or by the butlerbot, not by mergify.